<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="10246" started-at="2019-03-05T06:53:26Z" finished-at="2019-03-05T06:53:37Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="10246" started-at="2019-03-05T06:53:26Z" finished-at="2019-03-05T06:53:37Z">
      <class name="CI_CD_Demo.Test01">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:CI_CD_Demo.Test01@523884b2]" name="beforeClass" is-config="true" duration-ms="3769" started-at="2019-03-05T12:23:26Z" finished-at="2019-03-05T12:23:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="verifySearchButtonFail()[pri:0, instance:CI_CD_Demo.Test01@523884b2]" name="verifySearchButtonFail" duration-ms="4677" started-at="2019-03-05T12:23:30Z" finished-at="2019-03-05T12:23:35Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Text not found! expected [Google Fearch] but found [Google Search]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Text not found! expected [Google Fearch] but found [Google Search]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at CI_CD_Demo.Test01.verifySearchButtonFail(Test01.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchButtonFail -->
        <test-method status="PASS" signature="verifySearchButtonPass()[pri:0, instance:CI_CD_Demo.Test01@523884b2]" name="verifySearchButtonPass" duration-ms="614" started-at="2019-03-05T12:23:35Z" finished-at="2019-03-05T12:23:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchButtonPass -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:CI_CD_Demo.Test01@523884b2]" name="afterClass" is-config="true" duration-ms="1146" started-at="2019-03-05T12:23:35Z" finished-at="2019-03-05T12:23:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- CI_CD_Demo.Test01 -->
      <class name="CI_CD_Demo.Test02">
        <test-method status="FAIL" signature="verifySearchButtonFail()[pri:0, instance:CI_CD_Demo.Test02@5b275dab]" name="verifySearchButtonFail" duration-ms="3" started-at="2019-03-05T12:23:37Z" finished-at="2019-03-05T12:23:37Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at CI_CD_Demo.Test02.verifySearchButtonFail(Test02.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchButtonFail -->
      </class> <!-- CI_CD_Demo.Test02 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
